module-2/controllers/createUser.ts

```diff
@@ -0,0 +1,21 @@
+import { Request, Response } from 'express';
+import { StatusCodes } from 'http-status-codes';
+import { v4 as uuidv4 } from 'uuid';
+import { users } from '../data/users';
+import { errorBoundary } from '../helpers';
+
+const createUser = (req: Request, res: Response) => {
+  const { password, login, age } = req.body;
+  const newUser = {
+    id: uuidv4(),
+    login,
+    password,
+    age,
+  };
```

Comment:
```suggestion
  const newUser = {
    id: uuidv4(),
    ...req.body
  };
```

Reply:
Your suggestion was my first solution, but I changed it because what if req.body is polluted with other properties? Since I am not using an ORM, I only validate the data at the endpoint with Joi which only checks required fields, does not throw error if there are more fields on req.body, so any additional fields could also pass to the handler and hence end up saved in our data. So all in all, I think although more verbose, my solution is also more 'defensive'. 
What do you think?